modules/installation-mirror-repository.adoc

```diff
@@ -148,8 +150,8 @@ the `imageContentSources` data that you require when you install your cluster.
 
 ... Record the entire `imageContentSources` section from the output of the previous
 command. The information about your mirrors is unique to your mirrored repository, and you must add the `imageContentSources` section to the `install-config.yaml` file during installation.
-
-
+--
++
 [NOTE]
 ====
 The image name gets patched to Quay.io during the mirroring process, and the podman images will show Quay.io in the registry on the bootstrap virtual machine.
```

Comment:
```suggestion
Some image names, including the `podman` image, are patched to contain `quay.io` during the mirroring process and include `quay.io` in the registry.
```

I looked at source issue for this note, and I think that this might be more accurate. (You don't always have a bootstrap _vm_.)

Reply:
Now that I see the original issue was referring simply to the `podman images` command, I've dropped its mention as it's not really  necessary. Went with:

"Some image names are patched during the mirroring process to include `quay.io` as the source registry."